src/control/common/storage/commands.go

```diff
@@ -23,6 +23,13 @@
 
 package common_storage
 
+import (
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/common"
+	log "github.com/daos-stack/daos/src/control/logging"
```

Comment:
Not really happy about this. New code should not be using the global logger, as we want to get rid of it. Updating cmd/dmg to do proper logger injection is beyond the scope of this PR though. I've created DAOS-3255 for that work.